doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -247,6 +247,7 @@ Serial LTE modem
     They are replaced by the ``#XNRFCLOUDPOS`` command that allows to combine cellular and Wi-Fi data to determine the device location.
   * The AT commands ``#XAGPS`` and ``#XPGPS``.
```

Comment:
There is another mention about removing a part of #XSLMUART command. I think that it could be removed. We ended up removing everything.